tenable/io/v3/was/attachments/api.py

```diff
@@ -0,0 +1,68 @@
+'''
+Attachments
+===========
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`attachments <was-v2-attachments>` API.
+
+Methods available on ``tio.v3.was.attachments``:
+
+.. rst-class:: hide-signature
+.. autoclass:: AttachmentsAPI
+    :members:
+'''
+from io import BytesIO
+from typing import BinaryIO, Optional
+from uuid import UUID
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+
+
+class AttachmentsAPI(ExploreBaseEndpoint):
+
```

Comment:
'''
    This class contains methods related to Attachments API
    '''

Reply:
Done.